Run.ps1

```diff
@@ -0,0 +1,211 @@
+<#
+.SYNOPSIS
+
+Creates array of containers to generate load with Azure ACI.
+
+.DESCRIPTION
+
+When creating load tests you need docker, powershell 6.2+ and powershell AZ installed.
+See examples and documentation how tool can be used.
+
+Please don't run this on production Azure environment: script contain tear-down functionalities
+that remove resources automatically. There should be no risk anything else will be removed but
+authors of this tools will not take any responsibility if you run it on production environment.
+
+.EXAMPLE
+
+PS> ./Run.ps1 -Image "somelocalimage"
+
+.EXAMPLE
+
+PS> ./Run.ps1 -Image "somelocalimage" -TTLInMinutes 30 -Groups 3 -ContainerPerGroup 3
+
+#>
+[CmdLetBinding()]
+Param(
+
+    # Image name from local cache that contains load generator software.
+    # For example use 'example' app, build it locally with tag 'loadtest' and
+    # give parameter 'loadtest' for Image parameter. If you use image from external
+    # repository you must first pull it to local cache with 'docker pull'.
+    [Parameter(Mandatory)]
+    [ValidateNotNullOrEmpty()]
+    [string]
+    $Image,
+
+    # Generating load can require lots of resources and they are pretty expensive
+    # if they are not cleaned up correctly. For this reason tool has automatic tear-down
+    # functionality that removes all resources used automatically after this TTL period is over.
+    # Defaults to 60 minutes and can be set up to 6 hours.
+    [Parameter()]
+    [ValidateRange(5, 60*6)]
+    [int]
+    $TTLInMinutes = 60,
+
+    # Test setup is built from two parts: groups and containers. Group can host multiple
+    # containers. If kubernetes is familiar this concept is identical with 'pods'.
```

Comment:
> If you are familiar with Kubernates

or perhaps
> This concept is similar to Kubernates 'pods'

Reply:
Also "Kubernetes"

Reply:
I fixed that, that was my typo. Maybe I shouldn't copy paste 10 letter words.